Module.php

```diff
@@ -208,20 +208,8 @@ public function update()
 
     private function importCustomPagesDefaultTemplates(): bool
     {
-        $customPagesModule = Yii::$app->getModule('custom_pages');
-        if (!$customPagesModule ||
-            !$customPagesModule->isEnabled ||
-            !class_exists('humhub\modules\custom_pages\modules\template\services\ImportService')) {
-            return true;
-        }
-
-        $importService = new \humhub\modules\custom_pages\modules\template\services\ImportService();
-        if (!method_exists($importService, 'allowUpdateDefaultTemplates')) {
-            return true;
-        }
-
-        return $importService
-            ->allowUpdateDefaultTemplates()
-            ->importFromFolder(Yii::getAlias('@calendar/resources/custom-pages-templates'));
+        $importServiceClassName = '\humhub\modules\custom_pages\modules\template\services\ImportService';
+        return !method_exists($importServiceClassName, 'importDefaultTemplates') ||
```

Comment:
Why do we need this method exists? Isn't that always available with the event?

Reply:
@luke- If we will move it to some event, then yes we don't need the method checking, but as I have written [here](https://github.com/humhub/calendar/pull/545#discussion_r2063124800) I don't find such event, only if we will implement new core event.
I think it would be good to have an event which is always called after `MigrationService->migrate()` even if no migrations.
Do you agree? If yes, then min version of the modules must be increased, and what core branch can we touch, `master` or `develop`?